.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
I think this should be in your global gitignore file. Or edit `$repo_dir/info/exclude`